rp_asos/views.py

```diff
@@ -2,8 +2,25 @@
 from rest_framework import status
 from rest_framework.views import APIView
 
-from .tasks import patient_data_check, create_hospital_groups
+from .tasks import (
+    patient_data_check,
+    create_hospital_groups,
+    screening_record_check,
+)
 from rp_redcap.views import validate_project
+from sidekick.models import Organization
+
+
+def validate_organization(org_id, request):
```

Comment:
Nice function! Could we move this to `sidekick.utils.py`? I'd like to use it in other app views as well 🙏